README.md

```diff
@@ -1,250 +1,165 @@
 <img alt="Logo" width="196px" align="left" src="./contrib/logo/blue-green.png"></img>
 
-# `kcp` provides a true multi-tenant Kubernetes control plane for workloads on many clusters
+# kcp
 
-`kcp` is a generic [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRD) apiserver that is divided into multiple "[logical clusters](docs/investigations/logical-clusters.md)" that enable multitenancy of cluster-scoped resources such as CRDs and Namespaces. Each of these logical clusters is fully isolated from the others, allowing different teams, workloads, and use cases to live side by side.
+## Overview
+kcp is a Kubernetes-like control plane focusing on:
 
-<br clear="left"/>
+- A **control plane** for multiple independent, **isolated** “clusters” known as **workspaces**
+- Enabling API service providers to **offer APIs centrally** using **multi-tenant operators**
+- Easy **API consumption** for users in their workspaces
+- Flexible **scheduling** of workloads to physical clusters
+- **Transparent movement** of workloads among compatible physical clusters
+- **Advanced deployment strategies** for scenarios such as affinity/anti-affinity, geographic replication, cross-cloud replication, etc.
 
-![build status badge](https://github.com/kcp-dev/kcp/actions/workflows/ci.yaml/badge.svg)
+kcp can be a building block for SaaS service providers who need a **massively multi-tenant platform** to offer services to a large number of fully isolated tenants using Kubernetes-native APIs. This could hopefully be useful to cloud providers as well as enterprise IT departments offering APIs within their company.
```

Comment:
some other wording without the word "hopefully" maybe. "hopefully" sounds defensive, not convinced. Would formulate it more around "vision" or "goals".

Reply:
Maybe: "The goal is to be useful ...."

Reply:
Updated, PTAL